config/tracking_beam_test/tracking_beam_frame_to_ndisk.yaml

```diff
@@ -0,0 +1,77 @@
+##########################################
+#
+# tracking_beam_frame_to_ndisk.yaml
+#
+# Configuration for recording the tracking
+# beam to n disks.
+#
+# Author: Jing Luo
+#
+##########################################
+---
+type: config
+log_level: debug2
+cpu_affinity: [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15]
+samples_per_data_set: 49152
+num_pol: 2
+buffer_depth: 105
+sub_frames_per_merged_frame: 11
```

Comment:
Is there a reason this is `11`?   Those are still going to be small files.  I would go for something like `500` which should give ~50MB files, which will be more efficient to write.

Reply:
OK, it was just for the init test. Will change it